scene/resources/camera_texture.cpp

```diff
@@ -109,6 +127,7 @@ void CameraTexture::set_camera_active(bool p_active) {
 	if (feed.is_valid()) {
 		feed->set_active(p_active);
 		notify_property_list_changed();
+		call_deferred(SNAME("emit_changed"));
```

Comment:
Isn't `callable_mp((Resource *)this, &Resource::emit_changed).call_deferred();` the safer way? In this case we can call `_on_format_changed` here directly (maybe rename that function even).

Reply:
There's also the option of using `Resource::emit_changed` directly, which has some logic to decide when to defer or not:
```diff
void Resource::emit_changed() {
        if (ResourceLoader::is_within_load() && MessageQueue::get_main_singleton() != MessageQueue::get_singleton() && !MessageQueue::get_singleton()->is_flushing()) {
                // Let the connection happen on the call queue, later, since signals are not thread-safe.
                call_deferred("emit_signal", CoreStringName(changed));
        } else {
                emit_signal(CoreStringName(changed));
        }
}
```

Reply:
This sounds even better.

Reply:
This sounds even better.